Automattic/wp-calypso
# VideoPress Onboarding: Save title and site description when site is created

Fixes setting the site title and description (aka tagline) for the site. I created a new api request for creating the site and title, and setting the description via a separate api call after the site is created. There did not appear to be a way to set the description via the `sites/new` endpoint so it needed to be done separately.

**To Test**
 * Create a new site via `/start/videopress`.
 * Set a title and a description.
 * Complete your purchase.
 * Your site should have the title and tagline that you set!
